data/human/war reactions.txt

```diff
@@ -168,3 +167,70 @@ mission "War Reactions: Navy and Free Worlds Captains"
+			choice
+				`	"Break it up, kids."`
+				`	(Ignore them and keep walking.)`
+					decline
+			`	The three attackers turn around, looking furious. They eye your gun and run away. "Thanks. I'm Gus," says the fourth boy, as he slowly rises to his feet. He looks about twelve years old.`
+			`	Gus thinks for a second and looks worried. "Can you walk me to the factory? I'm already late for work, and I'm afraid they'll attack me again."`
+			choice
+				`	"Aren't you a little young to work in a factory?"`
+				`	"I'm in a hurry. You're on your own."`
+					decline
+			`	As you start walking, he answers. "No, I'm an orphan." He sees your confusion and explains. "The Syndicate lets some orphans work in factories, in return for better education, good housing, and medical care. Only the cleverest of us get to do it. All of the orphanage factories are safe; we don't use big dangerous equipment. Just ordinary stuff, like hammers, soldering irons, or sewing machines.`
+			`	"Anyway, you're a captain, right? Were you in the big fights? I was on Maker when I saw some Pug ships fighting human ships on our Syndicate school's big educational viewscreen."`
+			choice
+				`	"Yes, I'm Captain <first> <last>."`
+					goto me
+				`	"I blew up my share of Pug ships."`
+					goto blew
+				`	"Actually, I led the battle."`
+			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You're Captain <first> <last>! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand and you shake it.`
+				goto next
+			label me
+			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You led the attack! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand and you shake it.`
+			label blew
+			`	"I bet you did! I wish I was on one of our warships, bravely defend the human race. The honor, the glory, the mercenary pay. It sounds amazing."`
+				goto next
+			label next
+			`	"What was it like, fighting the evil Pug?"`
+
+			choice
+				`	"Actually, they're not evil. Their 'war' was a ploy to get us to stop fighting among ourselves. A lot of Pug gave their lives willingly to save us from self-destruction."`
+				`	"War is hell, kid. I'm not going to glorify it for you."`
+					goto hell
+				`	"Leading a battle is like chess with guns and engines. It's the most exhilarating feeling I've ever felt."`
+					goto chess
+				`	"I was an honor to protect the human race from our attackers."`
+					goto honor
+			`	"That's so insightful! Still, without people like you, the Pug's ploy would never have worked. Thank you for your service, captain."`
+				goto salute
+			label hell
+			`	"You went through hell for us, eh? That's something to think about. Maybe I should stick to engineering instead of becoming a combat pilot. I donno."`
```

Comment:
```suggestion
			`	"You went through hell for us, eh? That's something to think about. Maybe I should stick to engineering instead of becoming a combat pilot. I dunno."`
```